aws/policy/compute.yaml

```diff
@@ -33,6 +33,12 @@ Statement:
       - ec2:StopInstances
       - ec2:TerminateInstances
       - ec2:UnassignPrivateIpAddresses
+      - lightsail:DeleteInstance
+      - lightsail:GetInstance
+      - lightsail:GetInstances
+      - lightsail:RebootInstance
+      - lightsail:StartInstance
+      - lightsail:StopInstance
```

Comment:
My apologies @prasadkatti, these actions are actually in the wrong statement - I missed that this block has a condition of ec2:Region: us-east-1 as that line was collapsed by github.  This condition is not met when calling the lightsail API, these need to be moved to another statement.  I believe we should be able to move all of the lightsail actions to a single statement, however I'm getting test errors when testing with 63770 so I haven't fully tested yet.  Moving all these policies to the AllowLightsailCreateInstances statement I can confirm the terminator class appears to run successfully.

Reply:
I am sorry, I thought all the tests ran in us-east-1 which is why I added it to that statement. I can move all the lightsail actions to a single statement. 

I still don't fully understand what you mean by -

> This condition is not met when calling the lightsail API

Reply:
ohh, I get it! The condition only applies to EC2. :(

Reply:
Seems to be, I couldn't find it in the docs but the terminator class (when calling lightsail get_instances) gets an AccessDeniedException when this condition is applied.

Reply:
I found the docs. These are the only condition keys defined for lightsail - https://docs.aws.amazon.com/en_pv/IAM/latest/UserGuide/list_amazonlightsail.html#amazonlightsail-policy-keys

The `ec2:Region` condition key is EC2 Service level condition key.

Reply:
I have moved all lightsail actions to a single statement. Marking this as resolved.